js/src/common/states/ModalManagerState.js

```diff
@@ -11,22 +11,11 @@ export default class ModalManagerState {
    * @public
    */
   show(componentClass, attrs) {
-    // Breaking Change Compliance Warning, Remove in Beta 15.
     if (!(componentClass.prototype instanceof Modal)) {
       // This is duplicated so that if the error is caught, an error message still shows up in the debug console.
       console.error('The ModalManager can only show Modals');
       throw new Error('The ModalManager can only show Modals');
```

Comment:
Saves bytes and we're already editing this area anyway :)

```suggestion
      const invalidModalWarning = 'The ModalManager can only show Modals.';
      console.error(invalidModalWarning);
      throw new Error(invalidModalWarning);
```